src/components/books/Books.js

```diff
@@ -18,9 +18,8 @@ function Books() {
   const books = useSelector((state) => state.booksReducer);
 
   return (
-    <div>
-      <h2>Books</h2>
-      <ul>
+    <div className="flex flex-col justify-center items-center bg-gray-200">
+      <ul className="py-8 w-screen m-auto">
```

Comment:
As required, Styling is should be the same as the one provided on Zeplin

Your design

> ![Screenshot (362)](https://user-images.githubusercontent.com/66112760/150649701-021fe69f-d3a3-4096-910c-96c97da3583a.png)


Zeplin design

> ![Screenshot (364)](https://user-images.githubusercontent.com/66112760/150649796-a313f895-fa36-4a20-a09c-287ac311ef04.png)


Kindly work on the styling of the webpage 👍